[32m2022-10-27 15:49:29 +0000[0m - dagster - [34mDEBUG[0m - generate_trajectories - 12c7e573-d938-4707-b3e4-4e6c57e068ab - 6731 - collect_data - LOGS_CAPTURED - Started capturing logs for step: collect_data.
[32m2022-10-27 15:49:29 +0000[0m - dagster - [34mDEBUG[0m - generate_trajectories - 12c7e573-d938-4707-b3e4-4e6c57e068ab - 6731 - collect_data - STEP_START - Started execution of step "collect_data".
[32m2022-10-27 15:49:29 +0000[0m - dagster - [34mDEBUG[0m - generate_trajectories - 12c7e573-d938-4707-b3e4-4e6c57e068ab - 6731 - collect_data - LOADED_INPUT - Loaded input "interval" using input manager "io_manager", from output "result" of step "get_interval"
[32m2022-10-27 15:49:29 +0000[0m - dagster - [34mDEBUG[0m - generate_trajectories - 12c7e573-d938-4707-b3e4-4e6c57e068ab - 6731 - collect_data - STEP_INPUT - Got input "interval" of type "Tuple[datetime,datetime]". (Type check passed).
[32m2022-10-27 15:49:29 +0000[0m - dagster - [34mERROR[0m - [31mgenerate_trajectories - 12c7e573-d938-4707-b3e4-4e6c57e068ab - 6731 - collect_data - STEP_FAILURE - Execution of step "collect_data" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "collect_data"::

TypeError: 'NoneType' object is not subscriptable

Stack Trace:
  File "/usr/local/lib/python3.10/site-packages/dagster/_core/execution/plan/utils.py", line 47, in solid_execution_error_boundary
    yield
  File "/usr/local/lib/python3.10/site-packages/dagster/_utils/__init__.py", line 430, in iterate_with_context
    next_output = next(iterator)
  File "/usr/local/lib/python3.10/site-packages/dagster/_core/execution/plan/compute.py", line 109, in gen_from_async_gen
    yield loop.run_until_complete(async_gen.__anext__())
  File "/usr/local/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.10/site-packages/dagster/_core/execution/plan/compute_generator.py", line 68, in _coerce_async_solid_to_async_gen
    result = await awaitable
  File "/workspaces/pipeline-environment/dagster_ais/dagster_ais/ops/trajectory_ops.py", line 22, in collect_data
    psub = await js.subscribe("ais", durable="fff")
  File "/usr/local/lib/python3.10/site-packages/nats/js/client.py", line 202, in subscribe
    stream = await self._jsm.find_stream_name_by_subject(subject)
  File "/usr/local/lib/python3.10/site-packages/nats/js/manager.py", line 65, in find_stream_name_by_subject
    return info['streams'][0]
[0m
